docs/examples/ModestMapsExample/src/ModestMapsExample.ls

```diff
@@ -48,8 +48,8 @@ package
             //stage.reportFps = true;
             
             //create some providers
-            _mapProviders.pushSingle(new MapboxProvider(File.loadTextFile("assets/mapbox/id.txt"), File.loadTextFile("assets/mapbox/token.txt")));
             _mapProviders.pushSingle(new MicrosoftRoadMapProvider(true));
+            //_mapProviders.pushSingle(new MapboxProvider(File.loadTextFile("assets/mapbox/id.txt"), File.loadTextFile("assets/mapbox/token.txt")));

```

Comment:
Can you add something that loads the id, then prints a big warning/error if you haven't changed it? And obviously only loads the layer if so.